### PR TITLE
Gangams/troubelshoot doc updates

### PR DESCRIPTION
**Pull request by ganga1980**
- Updated README to cover acs-engine and brand name changes
- Updated aks troubleshoot scripts with fixes and improvements
- Fix the deployment length issue in CI solution template 
- Added separate troubleshooting script for ACS-engine

### CODE REVIEW

File: Troubleshoot/TroubleshootError_AcsEngine.ps1 [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+param(
+    [Parameter(mandatory = $true)]
+    [string]$SubscriptionId,
+    [Parameter(mandatory = $true)]
+    [string]$ResourceGroupName	
+)
+
+$ErrorActionPreference = "Stop";
+Start-Transcript -path .\TroubleshootDump.txt -Force
+$OptOutLink = "https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers"
```
Comment (bragi92) [inline Troubleshoot/TroubleshootError_AcsEngine.ps1]: Optional: https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers#uninstalling-the-chart

Comment (bragi92) [inline Troubleshoot/TroubleshootError_AcsEngine.ps1]: https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers#uninstalling-the-chart

File: Troubleshoot/TroubleshootError_AcsEngine.ps1 [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+$OptInLink = "https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers"
```
Comment (bragi92) [inline Troubleshoot/TroubleshootError_AcsEngine.ps1]: https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers#installing-the-chart

File: Troubleshoot/TroubleshootError_AcsEngine.ps1 [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+        Write-Host("")
+        Write-Host("Cannot fetch workspace location. Please try again...") -ForegroundColor Red
+        Write-Host("")
+        Stop-Transcript
+        exit
+    }
+
+    $WorkspacePricingTier = $WorkspaceInformation.sku
+
+    Write-Host("Pricing tier of the configured LogAnalytics workspace: '" + $WorkspacePricingTier + "' ") -ForegroundColor Green
```
Comment (bragi92) [inline Troubleshoot/TroubleshootError_AcsEngine.ps1]: Similar comment: Printing out the log analytics workspace id here.

File: Troubleshoot/TroubleshootError.ps1 [hunk 4]
```diff
@@ -166,198 +184,236 @@ if ($account.Account -eq $null) {
...
-		}
-	}
+    Write-Host("")
+    Write-Host("Could not fetch cluster details: Please make sure that the AKS Cluster name: '" + $AKSClusterName + "' is correct and you have access to the cluster") -ForegroundColor Red
+    Write-Host("")
+    Stop-Transcript
+    exit
+}
+else {
+    Write-Host("Successfully checked AKS Cluster details...") -ForegroundColor Green
```
Comment (bragi92) [inline Troubleshoot/TroubleshootError.ps1]: Printing the full cluster id will be helpful. Similarly for the log analytics workspace id.

Comment (ganga1980) [reply Troubleshoot/TroubleshootError.ps1]: > Printing the full cluster id will be helpful. Similarly for the log analytics workspace id.

Its there as well.

**Pull request merged**